### PR TITLE
Add anago to krel migration roadmap

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this:

/kind documentation

#### What this PR does / why we need it:
This moves over the Google Doc into a markdown format inside this
repository.

#### Which issue(s) this PR fixes:

None
#### Special notes for your reviewer:
Ref: https://docs.google.com/document/d/1FqHIMOd2NDQbP4X4mxWEjdZb9OFTPHxB8By6PCtsCGA/edit#

/cc @kubernetes/sig-release-leads

### CODE REVIEW

File: release-engineering/roadmaps/anago_krel_migration.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+# Krel - Anago Migration Roadmap
+
+_Moving away from running bash in production in k/release_
+
+## Objectives
+
+This roadmap defines a strategy for achieving two primary goals: migrating
+exchangeable bits of bash code within Anago to krel creating a Golang native
+replacement in krel that relies only on krel (stage/release) subcommands and
+enables us to remove anago completely.
```
Comment (hasheddan) [inline release-engineering/roadmaps/anago_krel_migration.md:10]: Looks like some sentences here got merged together

Comment (saschagrunert) [reply release-engineering/roadmaps/anago_krel_migration.md]: Yep, fixed that sentence. :+1:

**Comment by saschagrunert** [comment]
As discussed in the meeting today we'll propose it as a KEP in k/enhancements.

File: release-engineering/roadmaps/anago_krel_migration.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+  https://github.com/kubernetes/release/issues/1446 (TBD)
+
+All four issues can be worked on in parallel. This is not a comprehensive list:
+There are still parts in Anago that can be ported to bash and that are not part
+of any issue yet.
+
+#### Acceptance Criteria
+
+- All issues currently open will be resolved (#1534, #1536, #1446, #1459)
```
Comment (saschagrunert) [inline release-engineering/roadmaps/anago_krel_migration.md:65]: TODO: link issues correctly

**Pull request closed without merging**